bundle-workflow/tests/test_integ_workflow/integ_test/test_local_test_cluster.py

```diff
@@ -0,0 +1,104 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import subprocess
+import tempfile
+import unittest
+from unittest.mock import MagicMock, mock_open, patch
+
+import requests
+
+from manifests.bundle_manifest import BundleManifest
+from test_workflow.integ_test.local_test_cluster import LocalTestCluster
+from test_workflow.test_cluster import ClusterCreationException
+
+
+class MockResponse:
+    def __init__(self, text, status_code):
+        self.text = text
+        self.status_code = status_code
+
+
+class LocalTestClusterTests(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = None
+        self.data_path = os.path.realpath(
+            os.path.join(os.path.dirname(__file__), "../../tests_manifests/data")
+        )
+        self.manifest_filename = os.path.join(
+            self.data_path, "opensearch-bundle-1.1.0.yml"
+        )
+        self.manifest = BundleManifest.from_path(self.manifest_filename)
+        with tempfile.TemporaryDirectory() as work_dir:
```

Comment:
This does out of scope after `with` and creates a new directory. If you just need a dummy directory name you can do `tempfile.gettempdir()`